api/session.ts

```diff
@@ -0,0 +1,102 @@
+import { base } from "api/base"
+import { baseWithAuth } from "./base"
+import { UserType } from "api/types"
+import { errorHandler } from "api/utils"
+
+export interface LoginProps {
```

Comment:
Are you using this on another file? Why are you exporting these? Better to just move types to a types file so this'll be cleaner or just don't export if not.